src/Domain/Tags.hpp

```diff
@@ -44,7 +44,21 @@ template <size_t Dim, typename TargetFrame>
 struct DomainCreator {
   using type = std::unique_ptr<::DomainCreator<Dim, TargetFrame>>;
   static constexpr OptionString help = {"The domain to create initially"};
+
+  using const_global_cache_type = ::Domain<Dim, TargetFrame>;
+  template <typename Metavariables>
+  static const_global_cache_type convert_for_global_cache(
+      const type& option_data) noexcept {
+    return option_data->create_domain();
```

Comment:
Does this also allow for adding the initial extents somehow? It's needed by the domain initialization and (currently also) the DG initialization

Reply:
You could return a pair and try something with subitems to add them individually. That's all I can think of. We shouldn't be adding things only valid during initialization to the cache. @kidder requested the domain be available for AMR reasons.

Reply:
I don't think the initial extents should be in the GlobalCache, they should still be obtained the same way as now from a DomainCreator.  There are many reasons to have the Domain be in the GlobalCache

Reply:
yep, I understand now that this is for keeping the domain in the cache permanently, not for initialization :)